tests/services/test_crash_reports.py

```diff
@@ -0,0 +1,61 @@
+import glob
+import shutil
+import time
+
+import pytest
+
+from pymobiledevice3.services.crash_reports import CrashReportsManager
+
+
+@pytest.fixture(scope='function')
+def crash_manager(lockdown):
+    return CrashReportsManager(lockdown)
+
+
+def test_ls_default(crash_manager):
+    crash_manager.afc.makedirs('temp')
+    assert '/temp' in crash_manager.ls()
+
+
+def test_ls_path(crash_manager):
+    crash_manager.afc.makedirs('temp/temp')
+    assert '/temp/temp' in crash_manager.ls(path='/temp')
+
+
+@pytest.mark.parametrize('depth', [2, 3, 4])
+def test_ls_depth(crash_manager, depth):
+    path = ''
+    path_list = []
+    for i in range(depth):
+        path += f'/temp'
+        path_list.append(path)
+    crash_manager.afc.makedirs(path)
+    crash_list = crash_manager.ls(depth=depth)
+    for item in path_list:
+        assert item in crash_list
+
+
+def test_ls_depth_minus_one(crash_manager):
+    path_list = ['/temp', '/temp/temp', '/temp/temp/temp']
+    crash_manager.afc.makedirs(path_list[-1])
+    crash_list = crash_manager.ls(depth=-1)
+    for path in path_list:
+        assert path in crash_list
+
+
+def test_clear(crash_manager):
+    crash_manager.afc.makedirs('temp')
+    crash_manager.clear()
+    dir_list = crash_manager.ls()
+    assert 'temp' not in dir_list
+    for item in dir_list:
```

Comment:
Why?

Reply:
so we decided to check the creation date? @matan1008

Reply:
change test to check birth time of files that are not appstored